testapps/testapp/src/main/AndroidManifest.xml

```diff
@@ -43,7 +45,8 @@
         <activity
             android:name="com.microsoft.identity.client.testapp.MainActivity"
             android:theme="@style/AppTheme.NoActionBar"
-            android:windowSoftInputMode="stateHidden">
+            android:windowSoftInputMode="stateHidden"
+            android:taskAffinity="">
```

Comment:
Not sure we should change this by default, but need to choose how to test different manifest versions.

Reply:
Leaving as is.  I added a comment to explain this in the manifest.